docs/JestObjectAPI.md

```diff
@@ -989,6 +989,16 @@ This function is not available when using legacy fake timers implementation.
 
 :::
 
+### `jest.runToFrame()`
```

Comment:
Should I add these changes to `29.7` docs or just the base docs like I have now?

Reply:
Just base 👍